src/app/shared/date/timeago.pipe.ts

```diff
@@ -0,0 +1,35 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
```

Comment:
I'd like to see this moved out of Publish into the styleguide rather than have it be duplicated here.

Reply:
That makes sense! OK with you if I tackle that in next sprint? -> https://github.com/PRX/styleguide.prx.org/issues/80